TEST.md

```diff
@@ -194,6 +194,68 @@ To cleanup the namespaces after the test has finished, run:
 $ bin/test-cleanup
 ```
 
+### Testing the dashboard
+
+We use WebdriverIO to test how the web dashboard looks and operates locally in
+Chrome. For cross-browser testing, we use SauceLabs, which runs simulataneous
+tests on different browsers in the cloud.
+
+Whether you are running the tests locally or in the cloud, you'll first run
+`bin/web dev` which will start up the dashboard at `localhost:7777`.
+
+To run a local WebdriverIO instance that will run the tests on a local instance
+of Chrome, run:
+
+```bash
+$ bin/web integration local
```

Comment:
Because I'm not an astute reader, I missed the prior `bin/web dev` step and failed this command the first time. How about calling the previous command out more prominently:

Whether you are running the tests locally or in the cloud, you'll first start up the dashboard at `localhost:7777`. You can do that in one of two ways:

```bash
# standalone
bin/web run
```
OR
```bash
# with webpack
bin/web dev
```

Related, I typically omit the leading `$` if my `bash` block only contains commands and not command output, makes for easier cutting and pasting. I realize we're not consistent with this across our docs, so tioli.